services/web-server/src/index.js

```diff
@@ -20,18 +21,7 @@ const load = loader(
   {
     cfg: {
       requires: ['profile'],
-      setup: ({ profile }) => {
-        const cfg = config({ profile });
-
-        // apply some sanity-checks
-        assert(cfg.server.port, 'config server.port is required');
```

Comment:
these are not needed anymore?

Reply:
I moved them down to the server section, since those configs aren't set for writeDocs (and aren't needed in that case)